### DISCUSSION TITLE
.h files not found out of the box on Windows

### DISCUSSION THREAD

**Discussion by MihaMarkic** [BartmanAbyss/vscode-amiga-debug#239] [Q&A] [answered]
As in title, compiler whines that it can't find .h files. Any idea what to do?

**Comment by BartmanAbyss** [comment]
do you have a concrete example/screenshot?

**Comment by MihaMarkic** [comment]
Hi, I think I've been mislead by design time error messages about "can't find .h files".
In fact, this is compilation output (F5) on blank Amiga: Init Project

```
Compiling main.c
Compiling support/gcc8_c_support.c 
Assembling support/gcc8_a_support.s
Assembling support/depacker_doynax.s
D:/Temp/Amiga/First/main.c:506:1: fatal error: opening dependency file obj/main.d: No such file or directory
  506 | }
      | ^
compilation termAisnsaetmebdl.e
r messages:
Fatal error: can't create obj/gcc8_a_support.o: No such file or directory
D:/Temp/Amiga/First/support/gcc8_c_support.c:289:1: fatal error: opening dependency file obj/gcc8_c_support.d: No such file or directory
  289 | }
      | ^
compilation terminated.
gnumake: *** [Makefile:69: obj/main.o] Error 1
gnumake: *** Waiting for unfinished jobs....
gnumake: *** [Makefile:69: obj/gcc8_c_support.o] Error 1
gnumake: *** [Makefile:73: obj/gcc8_a_support.o] Error 1
```
I don't have much experience with C in general, though.

**Comment by MihaMarkic** [comment]
Duh, mystery solved. I had to manually create `out` and `obj` directories. I guess that could be handled by makefile?

**Comment by BartmanAbyss** [comment, marked as answer]
That‘s a temporary error with the latest version. These directories should have been created automatically. I have fixed it for the next version.
